src/types.ts

```diff
@@ -51,6 +51,12 @@ interface Lease {
   contractDate: Date | undefined
   lastDebitDate: Date | undefined
   approvalDate: Date | undefined
+  residentialArea?: ResidentialArea | undefined
```

Comment:
```suggestion
  residentialArea: ResidentialArea | undefined
```
`?`och `| undefined` har samma effekt i princip. Båda behövs inte